docs: minor heading fix

briancline opened the pull request:
Hello, I was just reading the docs and noticed a formatting issue on the identity config docs and figured I would send a quick fix. This PR includes two minor updates:
* LDAP heading was (visually) missing due to stray code block markup in the previous section.
* Minor casing change to Static provider heading for consistency with the others.

mhilton commented on idp/idptest/suite.go:
```diff
@@ -4,121 +4,104 @@
 package idptest
```
Perhaps rename the file fixture.go?

briancline committed:
docs: minor heading fix

LDAP heading was visually missing due to stray code block markup.
Minor casing change to static heading for consistency with others.

briancline committed:
Merge branch 'master' into minor-docs-fixes

alesstimec committed:
Merge branch 'master' into minor-docs-fixes
